stable/csi-secrets-store-provider-aws/README.md

```diff
@@ -1,5 +1,9 @@
 # csi-secrets-store-provider-aws
 
+**This Helm chart is deprecated, please switch to https://aws.github.io/secrets-store-csi-driver-provider-aws/ which is reviewed, owned and maintained by AWS.**
```

Comment:
Can you add this note to the main readme as well?

Reply:
Done